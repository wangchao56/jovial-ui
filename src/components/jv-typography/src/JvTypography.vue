<script setup lang="ts">
</script>

<template>
  <article class="jv-typography">
    <slot />
  </article>
</template>

<style lang="scss">
@use '@/theme/styles/typography' as *;
</style>
