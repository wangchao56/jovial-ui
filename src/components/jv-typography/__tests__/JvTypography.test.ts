import { mount } from '@vue/test-utils'
import { describe, expect, it } from 'vitest'
import JvTypography from '../src/JvTypography.vue'

describe('jvTypography', () => {
  it('renders correctly with default props', () => {
    const wrapper = mount(JvTypography)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
