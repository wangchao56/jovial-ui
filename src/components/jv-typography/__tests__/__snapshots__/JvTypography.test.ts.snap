// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`jvTypography > renders correctly with default props 1`] = `"<article class="jv-typography"></article>"`;
